### PR TITLE
[4.0] API Token

### PR DESCRIPTION
**Pull request by brianteeman**
This PR corrects the markup in the xml to match the existing standards. It also removes some classes that are not required and fixes the switcher.

I also changed the name from Joomla Token to API Token as I believe it to be a more accurate description but maybe @nikosdion and @wilsonge disagree. I am not precious about the name

To test make sure you have enabled the user - token plugin and that you check the plugin layout as well as the layout in the user profile

### before
image

### after
image

### before
image

### after
image

### CODE REVIEW

**Comment by wilsonge** [comment]
I'm ok if you wanna put API in the name. I kinda agree with Nic - that `API Token` on it's own probably isn't the best name - because it is plugin based and there may well be different plugins (especially because I assume there will be a community provided oAuth token provider that will happen for enterprise sites).

**Comment by brianteeman** [comment]
I have updated the PR to be " Joomla API token"

**Review by wilsonge — APPROVED** [review]

File: plugins/user/token/token/token.xml [hunk 1]
```diff
@@ -1,57 +1,66 @@
...
-				   label="PLG_USER_TOKEN_SAVEMEFOROTHERPEOPLE_DESC"
-				   heading="p"
-				   description=""
-				   close="0"
-				   class="alert alert-warning"
+			<field
+				name="savemeforotherpeople"
+				type="note"
+				label="PLG_USER_TOKEN_SAVEMEFOROTHERPEOPLE_DESC"
+				heading="p"
```
Comment (Quy) [inline plugins/user/token/token/token.xml:29]: Intentional removing `class="alert alert-warning"`?

Comment (brianteeman) [reply plugins/user/token/token/token.xml:29]: no it wasnt- thanks

File: plugins/user/token/token/token.xml [hunk 2]
```diff
@@ -1,57 +1,68 @@
...
+		<fieldset
+			name="joomlatoken"
+			label="PLG_USER_TOKEN_GROUP_LABEL"
+			description="PLG_USER_TOKEN_GROUP_DESC">
+
+			<field
+				name="saveme"
+				type="note"
+				label="PLG_USER_TOKEN_SAVEME_DESC"
+				heading="p"
```
Comment (Quy) [inline plugins/user/token/token/token.xml:13]:
```suggestion
				heading="p"
				class="alert alert-warning"
```

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 2002cb8cb9ddbf3228c834852394b4b792226f9b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28561">issues.joomla.org/tracker/joomla-cms/28561</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**